material-ui-superselectfield/material-ui-superselectfield.d.ts

```diff
@@ -0,0 +1,45 @@
+declare module "material-ui-superselectfield" {
+    import * as React from "react"
+
+    interface IMaterialUISuperSelectFieldProps {
+        name: string;
+        value: object[] | object | null;
+        multiple?: boolean;
+        key?: string;
+        onChange?: (...args: any[]) => void;
+        selectionsRenderer?: (...args: any[]) => void;
+        checkPosition?: string;
+        hintText?: string;
+        nb2show?: number;
+        elementHeight?: number;
+        style?: Object;
```

Comment:
Updated all objects to the more generic `Object` for flexibility in the future.  (Most of these appear to be used for styling purposes)